src/test/java/seedu/duke/database/DataBaseTest.java

```diff
@@ -0,0 +1,92 @@
+package seedu.duke.database;
+
+import java.io.FileNotFoundException;
+import java.util.NoSuchElementException;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Perhaps you would like to import specific methods rather than using wildcard import.